keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -50,6 +47,8 @@
     - [Problems](#problems)
 <!-- /toc -->
 
+# Object Storage Support
+
 # Release Signoff Checklist
 
 - [X] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
```

Comment:
are there any other boxes to check? Like maybe 2, 3, 4(mostly)?

Reply:
I wasn't focusing on making changes to the check boxes.  I was trying to narrowly focus to make the changes needed for sidecar.  Is there something that should be here or do in another PR?